dev/fattest.simplicity/test/componenttest/custom/junit/runner/SyntheticServletTests.java

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.custom.junit.runner;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+import jakarta.servlet.annotation.WebServlet;
+
+public class SyntheticServletTests {
+
+    public static class TestClass {
+        public static LibertyServer NOOP;
+    }
+
+    @WebServlet("*/")
+    public static class TestFilterByMethodServlet extends FATServlet {
+
+        @Mode(TestMode.FULL)
+        public void fullTest() {}
```

Comment:
Do all of the methods in these Servlets in this test need to also have `@Test`?

Reply:
They don't need to be.
The test is verifying the behavior of SyntheticServletTest, not of Junit itself.
The `@Test` annotation only serves to make these tests discoverable to the TestServletProcessor which we do not need to test SyntheticServletTest.